src/nvim/ui_bridge.c

```diff
@@ -172,6 +173,17 @@ static void ui_bridge_cursor_goto(UI *b, int row, int col)
 {
   UI_CALL(b, cursor_goto, 3, b, INT2PTR(row), INT2PTR(col));
 }
+
+static void ui_bridge_cursor_styleset(UI *b, Dictionary args)
+{
+  UI_CALL(b, cursor_styleset, 2, b, &args);
+}
+
+static void ui_bridge_cursor_styleset_event(void **argv)
```

Comment:
This should "hug" `ui_bridge_cursor_styleset`, instead of `ui_bridge_cursor_goto_event`. IOW delete the newline above, and add one below :)